debian/rules

```diff
@@ -139,14 +139,14 @@ override_dh_install: clean-config-vars prepare-scripts debian/config-dir/apache2
 
 override_dh_fixperms-arch:
 	# standard suexec
-	chmod 4754 debian/apache2-suexec-pristine/usr/lib/apache2/suexec-pristine
 	chgrp www-data debian/apache2-suexec-pristine/usr/lib/apache2/suexec-pristine
+	chmod 4754 debian/apache2-suexec-pristine/usr/lib/apache2/suexec-pristine
```

Comment:
this is hilarious, TIL chgrp removes the setuid bit

Reply:
Yeah I had no idea, apparently the chown system call (which chgrp calls internally) causes the kernel to remove the setuid/setuid bits as part of the process, so I think the Debian packaging is broken too. Not sure how they haven't noticed yet though, that seems strange.